Makefile

```diff
@@ -419,19 +429,15 @@ node-yarn-run:
 
 .PHONY: node-run
 node-run:
-ifneq ($(shell brew --prefix nvm 2>/dev/null),)
-	@echo "Using nvm from brew to select node version $(node_version)"
-	source $(shell brew --prefix nvm)/nvm.sh && nvm use $(node_version) && bash -c "$(node_cmd)"
-else ifneq ($(shell type nvm 2>/dev/null),)
+ifneq ($(shell command -v nvm 2>/dev/null),)
 	@echo "Using nvm to select node version $(node_version)"
 	nvm use $(node_version) && bash -c "$(node_cmd)"
 else
-	node --version 2>&1 >/dev/null || \
-		(echo "Trying apt install" && which apt-get && (curl -sL https://deb.nodesource.com/setup_$(node_version).x | bash) && apt-get install -y nodejs) || \
-		(echo "Trying apk install" && which apk && apk add --update nodejs nodejs-npm) || \
-		(echo "No node install or known package manager" && exit 1)
-	@echo "Not using nvm, using node version $(shell node --version)"
-	bash -c "$(node_cmd)"
+	mkdir .nvm
+	# Install nvm locally
+	curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | NVM_DIR=$$(pwd)/.nvm PROFILE=/dev/null bash
+	bash -c "source $$(pwd)/.nvm/nvm.sh; nvm install 6"
```

Comment:
hm would `SELF_DIR` (declared at the top of the makefile) be better than `pwd`?

Reply:
Ah nice catch, ty